main.go

```diff
@@ -10,7 +10,12 @@ type Proxy struct{}
 
 func NewProxy() *Proxy { return &Proxy{} }
 
+func enableCors(w *http.ResponseWriter) {
```

Comment:
I would do this in form of middleware.

Reply:
Removed HTTPS support since heroku seems to handle that, and refactored the handlers to a more middleware-like system.